dev/InfoBar/InfoBar.xaml

```diff
@@ -0,0 +1,219 @@
+                                    TextWrapping="WrapWholeWords"
+                                    FontWeight="{StaticResource InfoBarMessageFontWeight}"
+                                    FontSize="{StaticResource InfoBarMessageFontSize}"/>
+
+                                <ContentPresenter
+                                    Grid.Column="2"
+                                    primitives:InfoBarPanel.HorizontalMargin="{StaticResource InfoBarActionHorizontalLayoutMargin}"
+                                    primitives:InfoBarPanel.VerticalMargin="{StaticResource InfoBarActionVerticalLayoutMargin}"
+                                    VerticalAlignment="Top"
+                                    Content="{TemplateBinding ActionButton}">
+                                    
+                                    <ContentPresenter.Resources>
+                                        <Style TargetType="Button">
+                                            <Style.Setters>
+                                                <Setter Property="MinWidth" Value="{StaticResource InfoBarActionButtonMinWidth}"/>
+                                                <Setter Property="Height" Value="{StaticResource InfoBarActionButtonHeight}"/>
+                                                <Setter Property="Padding" Value="{StaticResource InfoBarActionButtonPadding}"/>
+                                                <contract7Present:Setter Property="CornerRadius" Value="{StaticResource InfoBarActionButtonCornerRadius}"/>
+                                            </Style.Setters>
+                                        </Style>
+
+                                        <Style TargetType="HyperlinkButton">
+                                            <Style.Setters>
+                                                <Setter Property="FontSize" Value="{ThemeResource InfoBarHyperlinkButtonFontSize}"/>
+                                                <Setter Property="Foreground" Value="{ThemeResource InfoBarHyperlinkButtonForeground}"/>
+                                                <Setter Property="Height" Value="{StaticResource InfoBarHyperlinkButtonHeight}"/>
+                                                <Setter Property="Padding" Value="{StaticResource InfoBarHyperlinkButtonPadding}"/>
+                                            </Style.Setters>
+                                        </Style>
+                                    </ContentPresenter.Resources>
+                                </ContentPresenter>
+
+                            </primitives:InfoBarPanel>
+
+                            <ContentPresenter
+                                Grid.Column="1"
+                                Grid.Row="1"
+                                Content="{TemplateBinding Content}"
+                                ContentTemplate="{TemplateBinding ContentTemplate}"/>
+
+                            <Button Name="CloseButton"
```

Comment:
Can we add a "Close" tooltip to the button? It would That would match the behavior of the TeachingTip and in general just provides better UX in my opinion.

![image](https://user-images.githubusercontent.com/1398851/96142975-ef0eae00-0f02-11eb-9752-1e17f352c68e.png)

(Initially posted this ask [here](https://github.com/microsoft/microsoft-ui-xaml/pull/3325#issuecomment-701343669) but I'm unsure if Github pings work when adding them as part of an edit of a message.)

Reply:
I didn't see this before, thanks for the new comment, @Felix-Dev. 
@teaP If this is a quick ask, let's include it in this PR -- otherwise we can add it in a different PR during this preview period.

Reply:
Added!